web/packages/teleport/src/components/Empty/Empty.tsx

```diff
@@ -71,48 +72,50 @@ export default function Empty(props: Props) {
       <Box maxWidth={600}>
         <Box mb={4} textAlign="center">
           <ResourceIcon name="server" mx="auto" mb={4} height="160px" />
-          <H1 mb={2}>{title}</H1>
-          <Text fontWeight={400} fontSize={14} style={{ opacity: '0.6' }}>
-            {byline}
+          <H1>{title}</H1>
+        </Box>
+        <Box mb={1} textAlign="center">
+          <H2>Automatic Discovery</H2>
+          <Text>
+            Connect your AWS, Azure, or GCP account to automatically scan and
+            import all resources.
+          </Text>
+        </Box>
+        <Box mb={4} textAlign="center">
+          <ButtonPrimary
+            as={Link}
+            to={cfg.getIntegrationsEnrollRoute({ tags: ['cloud'] })}
+            width="224px"
+          >
+            Connect Cloud Account
+          </ButtonPrimary>
+        </Box>
+        <Box mb={1} textAlign="center">
+          <H2>Manual Entry</H2>
+          <Text>
+            Browse and add individual servers, databases, or apps one at a time.
           </Text>
         </Box>
         <Box textAlign="center">
-          <Link
+          <ButtonPrimaryBorder
+            as={Link}
             to={{
-              pathname: `${cfg.routes.root}/discover`,
+              pathname: `${cfg.routes.discover}`,
               state: {
                 entity: 'unified_resource',
               },
             }}
-            style={{ textDecoration: 'none' }}
+            width="224px"
```

Comment:
we use `224px` twice to manage a similar width for multiple components. lets make it some `const`